tests/monotouch-test/MetalPerformanceShaders/MPSImageHistogramEqualizationTest.cs

```diff
@@ -20,17 +20,26 @@ namespace MonoTouchFixtures.MetalPerformanceShaders
 	[TestFixture]
 	public class MPSImageHistogramEqualizationTest
 	{
+		IMTLDevice device;
 
-		[Test]
-		public void Constructors ()
+		[TestFixtureSetUp]
+		public void Metal ()
 		{
 #if !MONOMAC
-			TestRuntime.AssertDevice ();
-			TestRuntime.AssertXcodeVersion (7, 0);
+			TestRuntime.AssertXcodeVersion (7,0);
 #else
-			TestRuntime.AssertXcodeVersion (9, 0);
+			TestRuntime.AssertXcodeVersion (9,0);
```

Comment:
Whitespace changes 😄 (same changes in all files)

BTW did VSM do this change? I find it a strange change to do

Reply:
no (or likely not). The `Metal` setup method was copied from other unit tests and it just _looks like_ whitespace changes in the diff.